### PR TITLE
Add 'disableClock' prop

### LINKED ISSUE #26: Unable to hide the clock
I'd like to be able to disable the clock, so it doesn't appear when editing the time.

However the `isClockOpen` prop doesn't seem to control this, as I thought it would; looking at the code, this doesn't appear to be a feature, unless I'm missing something?

If this is not currently a feature, nor in development (and is something you think would be a positive addition) please let me know - I'd be interested in trying to add this.

### PR DESCRIPTION
**Pull request by twinraven**
Closes #26.

I looked into adding a unit test for this, but I had issues running the test suite; once I got it running, some of the tests were failing already. If you'd like tests as part of this PR, would you be able to help me get it running? I suspect I'm missing something... thanks

### CODE REVIEW

**Comment by peduarte** [comment]
🎉

**Comment by twinraven** [comment]
Hi @wojtekmaj, do you have any feedback on this? Thanks

File: src/DateTimePicker.jsx [hunk 1]
```diff
@@ -346,6 +347,7 @@ DateTimePicker.defaultProps = {
   isCalendarOpen: null,
   isClockOpen: null,
   maxDetail: 'minute',
+  disableClock: false,
```
Comment (wojtekmaj) [inline src/DateTimePicker.jsx]: Isn't really necessary, non-existing prop is falsy

**Comment by wojtekmaj** [comment]
Generally I think it's good. Only thing, it's missing a unit test, but I can write it for you.

**Comment by wojtekmaj** [comment]
Thanks!

**Pull request merged**